src/translations/en.json

```
@@ -545,7 +545,7 @@
     "SubstationList": "Substation list",
     "NetworkModificationTree": "Network modification tree",
     "RestoreModifications": "Restore modifications",
-    "CreateCompositeModification": "Créer une modification réseau composée",
+    "CreateCompositeModification": "Create composite network modification",
```

I’m pretty much sure that an article is needed :
```suggestion
    "CreateCompositeModification": "Create a composite network modification",
```

---

I'm fine, but I prefer to respond to the PO request